packages/rocketchat-integrations/server/api/api.js

```diff
@@ -0,0 +1,358 @@
+/* globals Api Meteor Restivus logger processWebhookMessage*/
+// TODO: remove globals
+import vm from 'vm';
+import moment from 'moment';
+
+const compiledScripts = {};
+
+const buildSandbox = function(store = {}) {
+	const sandbox = {
+		_,
+		s,
+		console,
+		moment,
+		Store: {
+			set(key, val) {
+				return store[key] = val;
+			},
+			get(key) {
+				return store[key];
+			}
+		},
+		HTTP(method, url, options) {
+			try {
+				return {
+					result: HTTP.call(method, url, options)
+				};
+			} catch (error) {
+				return {
+					error
+				};
+			}
+		}
+	};
+
+	Object.keys(RocketChat.models).filter(function(k) {
```

Comment:
Can we turn these into single line arrow functions? Would clean this up quite a bit.